Makefile

```diff
@@ -83,7 +83,9 @@ init: default
 ### run:              Start the apisix server
 .PHONY: run
 run: default
-ifeq ("$(wildcard logs/nginx.pid)", "")
+ifeq ("$(findstring /root, ${PWD})", "/root")
+	@echo "Warning! It is forbidden to run APISIX in the /root directory."
```

Comment:
It should be an error and exit with nonzero code, as APISIX won't be run.

Reply:
updated.